### ISSUE TITLE
Modification time of non-existing tiles

### ISSUE THREAD

**Issue by dawinaj** [mapcrafter/mapcrafter#343]
After moving to new PC, i noticed probably unexpected behavior.
Output directory is newer than my worlds, and mapcrafter skips their generation, despite that tiles for these worlds don't even exist.

**Comment by elamperti** [comment]
This sounds like an issue associated with a particular setup rather than a mapcrafter problem (specially because there are no other cases).

It would help if you could share which OS you are using and Mapcrafter version.

**Comment by dawinaj** [comment]
Right. I was using Win10 with Mapcrafter for 1.13, looks like version was 2.3.1.
